src/ansiblelint/prerun.py

```diff
@@ -169,7 +169,7 @@ def _install_galaxy_role() -> None:
     role_namespace = yaml['galaxy_info'].get('namespace', None)
     if not role_namespace:
         role_namespace = yaml['galaxy_info'].get('author', None)
-    if not role_name:
+    if not role_name or os.path.exists(".git") or os.path.exists(".hg"):
```

Comment:
I am afraid that this may not be correct and it also seems unrelated, better to remove it from this patch. We can address it separately if needed.

Reply:
tbh, I've wrote this patch as I found the comment while trying to understand how things were supposed to work. I've not seen any bug about this so I've no problem with dropping this change.

Reply:
hm. forgot about dropping this. doing now